zhaquirks/philips/__init__.py

```diff
@@ -340,3 +345,21 @@ class PhilipsRwlRemoteCluster(PhilipsRemoteCluster):
         3: Button("down", DIM_DOWN),
         4: Button("off", TURN_OFF),
     }
+
+
+class PhilipsHueCluster(CustomCluster):
```

Comment:
Would renaming this to `PhilipsLightCluster` or `PhilipsHueLightCluster` make sense?